ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -9,12 +9,14 @@ import SwiftUI
 /// Properties that make up cell content
 @objc public class MSFListCellState: NSObject, ObservableObject, Identifiable {
     public var id = UUID()
-    @objc @Published public var leadingView: UIView?
+    @objc @Published public var leadingView: UIView? // Supporting square views only
+    @objc @Published public var isLargeIcon: Bool = false
     @objc @Published public var title: String = ""
     @objc @Published public var subtitle: String?
+    @objc @Published public var trailingView: UIView? // Supporting square views only
```

Comment:
I think the scenario we support in the trailingView is not necessarily just square views, right?
Any non-text-based view should still render correctly, I believe. It's just the label views that get compressed and disappear.

I think it's worth adding /// documentation here explaining more about the bug so that consumers of this code can be aware that it's a known issue.

Reply:
Resolved offline: leading/trailing views can only support views with the same width/height regardless of type